commit b3e0d0cb76b4713d801647bb408fe46e9a12d2b7
Author: Emil Edholm <bobby@verypublic.org>
Date:   Thu Feb 11 12:34:38 2016 +0100

    On Arch, some libraries need to be explicitly defined.
    
    The reason is that the binutils package does not allow
    dynamic linking of bfd for some reason or another.

diff --git a/Makefile b/Makefile
index 7e5283d..813f77e 100644
--- a/Makefile
+++ b/Makefile
@@ -39,7 +39,8 @@ ifeq ($(OS),Linux)
                    -D_FILE_OFFSET_BITS=64
     ARCH_LDFLAGS := -L/usr/local/include -L/usr/include \
                     -Bdynamic \
-                    -lpthread -lunwind-ptrace -lunwind-generic -lbfd -lopcodes -lrt
+                    -lpthread -lunwind-ptrace -lunwind-generic -lbfd -lopcodes -lrt \
+					-ldl -lz /usr/lib64/libiberty.a
     ARCH_SRCS := $(wildcard linux/*.c)
 
     ifeq ("$(wildcard /usr/include/bfd.h)","")
